lib/drush-node.js

```diff
@@ -1,5 +1,7 @@
+/* globals require, module, console */
```

Comment:
If this is a result of jshint stuff can you set [node](http://jshint.com/docs/options/#node) option in `.jshintrc`?